4_arm.ipynb

```diff
@@ -298,21 +546,45 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "手先座標は、3次元の位置と四元数（quaternion）による姿勢で表現されています。四元数の詳細が気になる方は調べてみてください。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The hand coordinates are represented by a 3D position and a quaternion. Please search on the internet, if you are interested in the details of the \"quaternion\"."
```

Comment:
"The hand coordinates are represented by a 3D position and a quaternion. Please search on the internet, if you are interested in details about \"quaternions\"."